admin/taxonomy/class-taxonomy-fields-presenter.php

```diff
@@ -44,11 +44,14 @@ public function html( array $fields ) {
 	private function form_row( $field_name, array $field_options ) {
 		$esc_field_name = esc_attr( $field_name );
 
-		$label = $this->get_label( $field_options['label'], $esc_field_name );
-		$field = $this->get_field( $field_options['type'], $esc_field_name, $this->get_field_value( $field_name ) , (array) $field_options['options'] );
-		$help  = $this->get_help( $field, $field_options['description'], $esc_field_name );
-
-		return $this->parse_row( $label, $help, $field );
+		$label            = $this->get_label( $field_options['label'], $esc_field_name );
+		$field            = $this->get_field( $field_options['type'], $esc_field_name, $this->get_field_value( $field_name ) , (array) $field_options['options'] );
+		$help_button_text = isset( $field_options['options']['help-button'] ) ? $field_options['options']['help-button'] : '';
+		$help             = new WPSEO_Admin_Help_Panel( $field_name, $help_button_text, $field_options['description'] );
+		$help_button      = $help->get_button_html();
+		$help_panel       = $help->get_panel_html();
+
+		return $this->parse_row( $label, $help_button, $help_panel, $field );
```

Comment:
Because you now have an object, you can pass an `WPSEO_Admin_Help_Panel` to this function instead of the two strings.